v2/apierror/api_error.go

```diff
@@ -27,7 +27,11 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-package gax
+/*
+Package apierror implements a wrapper error for parsing
+error details from gRPC calls.
```

Comment:
Nit: can this say `from API calls. Currently, only errors representing a gRPC Status are supported.`?